Source/Core/DiscIO/Volume.h

```diff
@@ -76,6 +76,12 @@ class Volume
   }
   virtual std::vector<u8> GetContent(u16 index) const { return {}; }
   virtual std::vector<u64> GetContentOffsets() const { return {}; }
+  virtual bool CheckContentIntegrity(const IOS::ES::Content& content, u64 content_offset,
+                                     const IOS::ES::TicketReader& ticket) const
+  {
+    return false;
+  }
+  virtual IOS::ES::TicketReader FixCommonKeyIndex() const { return {}; }
```

Comment:
Nitpick: this function's name implies it's fixing the ticket in-place, rather than returning a new, fixed ticket. Maybe GetTicketWithFixedCommonKeyIndex, or GetFixedTicket if that's too long?